### ISSUE TITLE
ONNX and TensorRT conversion

### ISSUE THREAD

**Issue by namKolorfuL** [facebookresearch/pytorchvideo#184]
Hi, I'm trying to convert a SlowFast pytorchvideo model to ONNX and TensorRT to get faster real time speed on embedded platforms, does pytorchvideo currently support this? If not, how should I write a convert script for ONNX? Thank you!

**Comment by MGReyes** [comment]
Hi, I'm trying to do the same with the X3D_S model! My goal is to convert the model to OpenVINO. The first step is to convert it to ONNX. When I try it I get the error:
`RuntimeError: ONNX export failed: Couldn't export Python operator SwishFunction`

**Comment by namKolorfuL** [comment]
@MGReyes seems like this repo doesn't support exportable version of SwishFunction, see this [https://github.com/lukemelas/EfficientNet-PyTorch/blob/master/efficientnet_pytorch/utils.py](url) and [https://github.com/lukemelas/EfficientNet-PyTorch/blob/master/efficientnet_pytorch/model.py](url) where they implemented a swish function for export.

**Comment by namKolorfuL** [comment]
@MGReyes or you can change swish function to RELU or sth [https://github.com/dusty-nv/jetson-inference/issues/1199](url)

**Comment by namKolorfuL** [comment]
@MGReyes would you mind sharing your onnx conversion script?

**Comment by hongsamvo** [comment]
@namKolorfuL I replaced SwishFunction by nn.SiLU and convert successfully, but my trt model is slower than torch model about 4 time. Did you convert successfully, how about speed of your trt model or onnx model?

**Comment by namKolorfuL** [comment]
@hongsamvo did you convert and infer the model on the same jetson/pc? It has to be on the same device or the performance will suck. Btw, did you add fp16 flag during the trt conversion?

**Comment by hongsamvo** [comment]
@namKolorfuL yes, I convert and infer x3dm on the same jetson NX, pytorch take ~70-80ms, and trt take ~380ms, I have tried fp16 but I don't see the improvement.
